tests/php/sync/test_class.jetpack-sync-actions.php

```diff
@@ -172,4 +172,14 @@ public function test_do_cron_sync_by_type_full_sync_send_lock() {
 		( new Lock() )->remove( 'full_sync' );
 	}
 
+	/**
+	 * Validate that mark_sync_read_only sets JETPACK_SYNC_READ_ONLY constant.
+	 */
+	public function test_mark_sync_read_only() {
+		$this->assertFalse( \Automattic\Jetpack\Constants::is_true( 'JETPACK_SYNC_READ_ONLY' ) );
+
+		Actions::mark_sync_read_only();
+		$this->assertTrue( \Automattic\Jetpack\Constants::is_true( 'JETPACK_SYNC_READ_ONLY' ) );
```

Comment:
I took a quick look at the failing unit tests, and it looks like they're caused by the `JETPACK_SYNC_READ_ONLY` constant being set in this test. I think they can be fixed by clearing the `JETPACK_SYNC_READ_ONLY` constant after this test.

Reply:
good eye! :( shame on me for not clearing it like in the other unit test.